config/core/test-configs.yaml

```diff
@@ -2227,6 +2234,7 @@ test_configs:
       - kselftest-filesystems
       - kselftest-futex
       - kselftest-lib
+      - kselftest-livepatch
```

Comment:
This should be fine, there are 10 `hana` Chromebooks online in Collabora's lab and they're not used by any other project. We could even try to run every possible test plan on them, until we start reaching a capacity limit.